server.js

```diff
@@ -6,6 +6,11 @@ var formidable = require('formidable'),
 
 var DIRECT_UPLOAD_LIMIT = 32; // bytes
 
+// convert from UTF-8 to ISO-8859-1
+var LATIN1_SYMBOLS = '¥§©ÆÖÑøøø¼';

```

Comment:
Yeah, you're right. They're both equally difficult to maintain. I guess we'll just wait and see, and if it becomes a problem we can revisit it later.